package.json

```diff
@@ -113,15 +114,20 @@
   "dependencies": {
     "@mdx-js/loader": "^2.3.0",
     "@types/dompurify": "^2.4.0",
-    "dompurify": "^2.4.3",
+    "d3": "^7.9.0",
+    "dompurify": "^3.1.6",
+    "esri-leaflet": "^3.0.12",
     "file-loader": "^6.2.0",
     "fitty": "^2.3.6",
     "gsap": "^3.11.3",
+    "leaflet": "^1.9.4",
     "postcss": "^8.4.31",
-    "preact": "10.22.0",
```

Comment:
The preact version has been downgraded. I imagine this is just a merge issue, can you try updating the PR to use 10.22.0? (beyond that version I think we may have build issues)